### ISSUE TITLE
Websocket time limit?

### ISSUE THREAD

**Issue by surfingreg** [finnhubio/Finnhub-API#517]
Hi, new to Finnhub, thanks for the work! Is there a time limit on the websocket connection? I'm seeing it hang after a few hours, though not a regular pattern. My only indication so far is the data stops coming. I'm still troubleshooting but the Alpaca websocket has remained up throughout so it hasn't been my internet dropping or anything. Figured I'd ask about a time limit first.... Thanks. 

Update 6/8/2023:
In case it corresponds to anything on the Finnhub end, I stopped receiving pings at 
2023-06-08 08:59:03.504603 UTC
2023-06-08 19:59:25.276077 UTC

The second time was after running only 40 minutes. The first one was at 2am my time. Both times when it happened the other websocket I have running (over the internet, using the same Rust code) did not stop pinging. Am I missing any published expectations either way as to how long the connection should last assuming the network isn't interrupted? Should I be coding in preemptive restarts? 

Update 6/9/2023:
Connection dropped again at:
2023-06-09 08:59:02.985189 UTC

...which leads me to believe (the free tier at least) websocket gets disconnected periodically, by design. Kinda seems like crickets here so I'll try emailing support as well.

**Comment by ValeryStatinov** [comment]
@surfingreg hello! have you received any additional info from the support?

**Comment by surfingreg** [comment]
No, never. So I assumed they were shutting down and removed them from my app.

**Comment by ValeryStatinov** [comment]
@surfingreg thanks for your reply!
